src/overlays/actors/ovl_Bg_Last_Bwall/z_bg_last_bwall.c

```diff
@@ -28,57 +37,210 @@ const ActorInit Bg_Last_Bwall_InitVars = {
     (ActorFunc)BgLastBwall_Draw,
 };
 
-// static ColliderTrisElementInit sTrisElementsInit[2] = {
-static ColliderTrisElementInit D_80C189C0[2] = {
+static ColliderTrisElementInit D_80C189C0[] = {
     {
-        { ELEMTYPE_UNK0, { 0x00000000, 0x00, 0x00 }, { 0x00000008, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_NONE, },
+        {
+            ELEMTYPE_UNK0,
+            { 0x00000000, 0x00, 0x00 },
+            { 0x00000008, 0x00, 0x00 },
+            TOUCH_NONE | TOUCH_SFX_NORMAL,
+            BUMP_ON,
+            OCELEM_NONE,
+        },
         { { { -70.0f, 0.0f, 3.0f }, { 70.0f, 0.0f, 3.0f }, { -70.0f, 200.0f, 3.0f } } },
     },
     {
-        { ELEMTYPE_UNK0, { 0x00000000, 0x00, 0x00 }, { 0x00000008, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_NONE, },
+        {
+            ELEMTYPE_UNK0,
+            { 0x00000000, 0x00, 0x00 },
+            { 0x00000008, 0x00, 0x00 },
+            TOUCH_NONE | TOUCH_SFX_NORMAL,
+            BUMP_ON,
+            OCELEM_NONE,
+        },
         { { { 70.0f, 0.0f, 3.0f }, { 70.0f, 200.0f, 3.0f }, { -70.0f, 200.0f, 3.0f } } },
     },
 };
 
-// static ColliderTrisInit sTrisInit = {
 static ColliderTrisInit D_80C18A38 = {
-    { COLTYPE_NONE, AT_NONE, AC_ON | AC_TYPE_PLAYER, OC1_NONE, OC2_NONE, COLSHAPE_TRIS, },
-    ARRAY_COUNT(sTrisElementsInit), D_80C189C0, // sTrisElementsInit,
-};
-
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_80C18AC8[] = {
-    ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_STOP),
+    {
+        COLTYPE_NONE,
+        AT_NONE,
+        AC_ON | AC_TYPE_PLAYER,
+        OC1_NONE,
+        OC2_NONE,
+        COLSHAPE_TRIS,
+    },
+    ARRAY_COUNT(D_80C189C0),
+    D_80C189C0, // sTrisElementsInit,
 };
 
-#endif
-
-extern ColliderTrisElementInit D_80C189C0[2];
-extern ColliderTrisInit D_80C18A38;
-extern InitChainEntry D_80C18AC8[];
-
-extern UNK_TYPE D_06000098;
+static ModelInfo D_80C18A48[] = { { object_last_obj_DL_0001A8, 0x0, &object_last_obj_Colheader_000288 },
+                                  { object_last_obj_DL_001318, 0x0, &object_last_obj_Colheader_001450 } };
```

Comment:
```suggestion
                                  { object_last_obj_DL_001318, 0x0, &object_last_obj_Colheader_001450 }, };
```
Should format better with a trailing comma. 
Side note, This is completely up to you, but based on other similar structs, the middle element is probably just another pointer of some kind, so could type the member as `void*` then use `NULL` here instead of `0x0`.